MekHQ/src/mekhq/gui/dialog/nagDialogs/NoCommanderNagDialog.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 - The MegaMek Team. All Rights Reserved.
```

Comment:
hehe might make this 2024

Reply:
Yeah, that would be helpful wouldn't it? Fixed,